CMakeLists.txt

```diff
@@ -109,21 +124,24 @@ find_package(Qhull)
 
 
 
-find_package(Boost COMPONENTS thread system chrono ${boost_python} ${boost_numpy})
+find_package(Boost 1.69 COMPONENTS thread system chrono ${boost_python} ${boost_numpy})
```

Comment:
@fredboudon : the required boost version is 1.69 while it is up to 1.78 in meta.yml

Reply:
in cmake this is the minimal version.